help.html

```diff
@@ -261,10 +290,12 @@ <h2 id="hello">Hello, World!</h2>
 
     <p>Here's how we do this in the Python editor:</p>
 
-    <pre><code class="python">from microbit import *
-
-
-display.scroll("Hello, World!")</code></pre>
+    <pre>
+        <code class="python">
+        from microbit import *
+        display.scroll("Hello, World!")
+        </code>
+    </pre>
```

Comment:
Is this rendering normal? 
![image](https://user-images.githubusercontent.com/29712657/62537899-72d20280-b849-11e9-9248-a17f86bb8cc7.png)
I think the spaces and new lines within `pre`are giving it a large margin.

Reply:
Same with the other <pre> tags.

Reply:
fixed in https://github.com/bbcmicrobit/PythonEditor/pull/199/commits/d78d2bdff9d61c165df613e35fcf18347e544078